### PR TITLE
Feature 3 add new routes for media format types

### PR DESCRIPTION
**Pull request by Jdallos**
-new routes for media types
-new layout dialog creates layouts with html media format

### CODE REVIEW

File: src/components/screens/layouts-screen.tsx [hunk 1]
```diff
@@ -161,7 +162,9 @@ class LayoutsScreen extends React.Component<Props, State> {
 
     const layoutCards = layouts.map(layout => {
       const layoutId = layout.id;
-      if (!layoutId) {
+      const layoutType = layout.layoutType;
```
Comment (EVP1989) [inline src/components/screens/layouts-screen.tsx:165]: Should we convert this to a functional component?

File: src/components/generic/content-map.tsx [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import * as React from "react";
 import { Map as LeafletMap, ImageOverlay, ScaleControl } from "react-leaflet";
 import { Map as MapInstance, LatLngBounds, CRS, LatLng, LeafletMouseEvent, Layer, MarkerOptions, LatLngExpression, LatLngTuple } from "leaflet";
-// TODO: Code mirror related imports.
-// import 'leaflet/dist/leaflet.css';
+import 'leaflet/dist/leaflet.css';
 import L from "leaflet";
 import "leaflet-draw";
-// import "leaflet-draw/dist/leaflet.draw.css";
+import "leaflet-draw/dist/leaflet.draw.css";
```
Comment (EVP1989) [inline src/components/generic/content-map.tsx:7]: Should this be converted to a functional component?

File: src/components/generic/confirm-dialog.tsx [hunk 3]
```diff
@@ -42,38 +42,39 @@ export default class ConfirmDialog extends React.Component<Props, State> {
 
     const dialogText = confirmDialogData.deletePossible ? confirmDialogData.text : confirmDialogData.contentTitle;
 
-    return
+    return (
       <>
-      <Dialog
```
Comment (EVP1989) [inline src/components/generic/confirm-dialog.tsx:47]: Should this be converted to a functional component?

**Review by EVP1989 — COMMENTED** [review]
Looks good but was the idea to convert everything we touch to a functional component?

File: src/components/generic/confirm-dialog.tsx (hunk 3 above)
Comment (Jdallos) [reply src/components/generic/confirm-dialog.tsx:47]: Discussed with Ville, initially decided not to do this, not it is being done. Expect new commit from Ville soon

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**